_posts/05-06-01-Internationalization-and-Localization.md

```diff
@@ -328,48 +328,48 @@ is usually your templates folder(s)
 translations, based on other ones you did. Those guesses and the changed entries will receive a "Fuzzy" marker,
-indicating it needs review, being highlighted in the list. It's also useful if you have a translation team and someone
-tries to write something they're not sure about: just mark Fuzzy and someone else will review later.
+indicating it needs review, being highlighted in the list. It is also useful if you have a translation team and someone
+tries to write something they are not sure about: just mark Fuzzy, and someone else will review later.
 
-Finally, it's advised to leave "View > Untranslated entries first" marked, as it will help you _a lot_ to not forget
+Finally, it is advised to leave "View > Untranslated entries first" marked, as it will help you _a lot_ to not forget
 any entry. From that menu, you can also open parts of the UI that allow you to leave contextual information for
 translators if needed.
 
 ### Tips & Tricks
 
 #### Possible caching issues
-If you're running PHP as a module on Apache (`mod_php`), you might face issues with the `.mo` file being cached. It
-happens the first time it's read, and then, to update it, you might need to restart the server. On Nginx and PHP5 it
+If you are running PHP as a module on Apache (`mod_php`), you might face issues with the `.mo` file being cached. It
+happens the first time it is read, and then, to update it, you might need to restart the server. On Nginx and PHP5 it
 usually takes only a couple of page refreshes to refresh the translation cache, and on PHP7 it is rarely needed.
 
 #### Additional helper functions
-As preferred by many people, it's easier to use `_()` instead of `gettext()`. Many custom i18n libraries from
-frameworks use something similar to `t()` as well, to make translated code shorter. However, that's the only function
+As preferred by many people, it is easier to use `_()` instead of `gettext()`. Many custom i18n libraries from
+frameworks use something similar to `t()` as well, to make translated code shorter. However, that is the only function
 that sports a shortcut. You might want to add in your project some others, such as `__()` or `_n()` for `ngettext()`,
 or maybe a fancy `_r()` that would join `gettext()` and `sprintf()` calls. Other libraries, such as
 [oscarotero's Gettext][oscarotero] also provide helper functions like these.
 
 In those cases, you'll need to instruct the Gettext utility on how to extract the strings from those new functions.
-Don't be afraid, it's very easy. It's just a field in the `.po` file, or a Settings screen on Poedit. In the editor,
+Don't be afraid; it is very easy. It is just a field in the `.po` file, or a Settings screen on Poedit. In the editor,
 that option is inside "Catalog > Properties > Source keywords". You need to include there the specifications of those
-new functions, following [a specific format](func_format):
+new functions, following [a specific format][func_format]:
```

Comment:
I expect that this will break the link - the expected format is `[link title](link ref or url)`
https://guides.github.com/features/mastering-markdown/